sechub-pds-solutions/owasp-zap/docker/Owasp-Zap-Debian.dockerfile

```diff
@@ -4,62 +4,37 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
+LABEL org.opencontainers.image.source="https://github.com/mercedes-benz/sechub"
+LABEL org.opencontainers.image.title="SecHub GoSec+PDS Image"
```

Comment:
```suggestion
LABEL org.opencontainers.image.title="SecHub OWASP ZAP+PDS Image"
```

Reply:
Ooooh - How could I miss that one?
Fixed.